### PR TITLE
Update README with run_pipeline.sh; deleted 0_create_uuid_table.sh

### PR DESCRIPTION
**Pull request by as2388**
Requires #73, #47.

### CODE REVIEW

**Comment by lukechurch** [comment]
@as2388 Could you fix the merge conflicts here? Given it's a UUID table PR, I'd like to be a bit super cautious

**Comment by as2388** [comment]
@lukechurch fixed merge conflicts

File: README.md [hunk 1]
```diff
@@ -15,31 +15,45 @@ Development requires the following additional tools:
...
-$ ./0_create_uuid_table.sh <data-root> 
+$ ./run_pipeline.sh <user> <pipeline-configuration-file-path> <coda-pull-auth-file> <coda-push-auth-file> <google-cloud-credentials-file-path> <coda-tools-root> <data-root> <data-backup-dir>
 ```
 
-where `data-root` is an absolute path to the directory in which all pipeline data should be stored. 
-The UUID table will be saved to a file in the directory `<data-root>/UUIDs`.
+where:
+- `user` is the identifier of the person running the script, for use in the TracedData Metadata 
+  e.g. `user@africasvoices.org` 
+- `pipeline-configuration-file-path ` is an absolute path to a pipeline configuration json file.
```
Comment (lukechurch) [inline README.md]: nit: space after '-file-path'

**Review by lukechurch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (as2388) [reply README.md]: fixed

**Pull request merged**